src/VisualStudio/NuGet.Packaging.VisualStudio/Templates/Projects/Package/Package.nuproj

```diff
@@ -13,8 +13,9 @@
 		</ProjectConfiguration>
 	</ItemGroup>
 	<PropertyGroup Label="Configuration">
-		<PackageId>$safeprojectname$</PackageId>
-		<Title>$projectname$</Title>
+    <PackageId>$safeprojectname$</PackageId>
+    <PackageVersion>1.0</PackageVersion>
```

Comment:
We should not provide a static version #. The [unfolded .targets](https://github.com/NuGet/NuGet.Build.Packaging/blob/dev/src/VisualStudio/NuGet.Packaging.VisualStudio/Templates/Projects/Package/Package.targets#L4-L9) already showcase how to create one dynamically